howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/index.js

```diff
@@ -0,0 +1,63 @@
+let conversation;
+
+Talk.ready.then(() => {
+  const me = new Talk.User({
```

Comment:
It was slightly confusing to me to see the inbox from the point of view of the hidden user, I was expecting to see it from the point of view of a normal user, so it was clear that one participant was missing. So I'd suggest swapping them round

Reply:
Got it

Reply:
Thanks, that does look clearer to me!